package.json

```diff
@@ -886,9 +896,12 @@
       ],
       "explorer": [
         {
-          "id": "powerpages.treeWebView",
-          "name": "Power Pages Actions",
-          "when": "isWeb && config.powerPlatform.experimental.enableCoPresenceFeature"
+          "id": "powerpages.powerPagesFileExplorer",
+          "name": "POWER PAGES ACTIONS",
```

Comment:
Name should be localized like the `pacCLI.authPanel` name above.
Looks like we missed that for the "Copilot In Power Page" -- that should be localized as well

Reply:
updated  - will ask Amit to address co-pilot one in a differen PR - if thats needed  -given we are only available in US geo - i beleieve that is intentional.